notebook/nbextensions.py

```diff
@@ -148,6 +148,10 @@ def install_nbextension(path, overwrite=False, symlink=False, user=False, prefix
     # make sure nbextensions dir exists
     ensure_dir_exists(nbext)
     
+    # forcing symlink parameter to False if os.symlink does not exist (e.g., on Windows machines running python 2)
+    if 'symlink' not in dir(os):
```

Comment:
I think the normal check for this would be `if not hasattr(os, 'symlink')`, which avoids computing the whole list and then searching the whole list for 'symlink'.